Steer/DigitizerWorkflow/src/TRDDigitWriterSpec.h

```diff
@@ -31,7 +31,6 @@ o2::framework::DataProcessorSpec getTRDDigitWriterSpec()
   return MakeRootTreeWriterSpec("TRDDigitWriter",
                                 "trddigits.root",
                                 "o2sim",
-                                1,
```

Comment:
what are the consequences of this? As far as I know, the workflow will now **never** exit?
I believe it is better to change the behaviour in the MakeRootTreeWriterSpec to use   `pc.services().get<ControlService>().readyToQuit(false);`

Reply:
Yes, this I will do as well.

Concerning the default value I was not aware of the fact that most of the processes only run once and then terminate. If this is the current behaviour of the workflow the default is ok.
I thought there is another "master" instance terminating the workflow, but I have checked the code.